### PR TITLE
Adding CondaNSIS

### PR DESCRIPTION
**Pull request by guilhermebs**
@conda-forge/help-python

Checklist
- [X] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [X] Source is from official source.
- [X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [X] If static libraries are linked in, the license of the static library is packaged.
- [X] Build number is 0.
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [X] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [X] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by guilhermebs** [comment]
@conda-forge-admin, please ping team

File: recipes/CondaNSIS/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 0e44381750b27041e1976f26ac04cb66503a933970d8c52208ac3fa43d8551a3
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
```
Comment (beckermr) [inline recipes/CondaNSIS/meta.yaml:18]: this should be host, not build

Comment (guilhermebs) [reply recipes/CondaNSIS/meta.yaml]: Thanks for the review! It's now been changed to host.

Also, the osx test is failing due to a missing dependency.

**Comment by dopplershift** [comment]
osx was failing with:
> ImportError: cannot import name 'gcd' from 'fractions'

from networkX--so not really a missing dependency. Closed & re-opened to re-run CI and hopefully get a clean build.

**Comment by jakirkham** [comment]
cc @anaconda-pkg-build (in case this is of interest to you as well)

**Comment by guilhermebs** [comment]
Thanks, now it seems to give the "correct" error message

[`conda_build.exceptions.DependencyNeedsBuildingError: Unsatisfiable dependencies for platform osx-64: {'nsis'}
`](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=309336&view=logs&j=e35d4f76-8ff2-5536-d795-df91e63eb9f7&t=fa7b4b17-b6ff-5c9c-8cfc-15f888c92310&l=1032)

As you can see the [NSIS conda-forge package](https://anaconda.org/conda-forge/nsis) does not support OSX, so this is expected

**Review by dopplershift — APPROVED** [review]

**Pull request merged**